src/components/StandaloneHeader.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React from "react";
+import { StyleSheet } from "react-native";
+import Header from "./Header";
+import ContentPadding from "./ContentPadding";
+import Text from "./Text";
+import { whiteColor, lightNavyBlueColor } from "../constants/colors";
+
+export type Props = {
+  title: string
+};
+
+const StandaloneHeader = ({ title }: Props) => (
```

Comment:
I suggest just inlining this into the SavedEventsScreen for now. Once this screen is in master we can then look at all of the headers and make the appropriate abstraction.

Reply:
Yeah, not that seriously for the duplication problem. I still think we should not abstract just yet as this kind of header appears in about 5 places currently. I think we should just address that all in one PR to make it clear as to the approach.

Reply:
I've reverted this now :)